airbyte-cdk/python/airbyte_cdk/sources/declarative/manifest_declarative_source.py

```diff
@@ -81,6 +81,9 @@ def __init__(self, source_config: ConnectionDefinition, debug: bool = False, con
         if unknown_fields:
             raise InvalidConnectorDefinitionException(f"Found unknown top-level fields: {unknown_fields}")
 
+    def resolved_manifest(self) -> Mapping[str, Any]:
```

Comment:
nit: I don't know about the team conventions: should this be a `@property`? If not, should it be rename using an action verb like `resolve_manifest`?

Reply:
I like the idea of making it an `@property`, will do that (and so will keep it as a noun).